Google.PowerShell.DotnetCore/Google.PowerShell.DotnetCore.csproj

```diff
@@ -21,7 +21,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="..\Google.PowerShell\Common\*.cs;..\Google.PowerShell\Compute\*.cs;..\Google.PowerShell\Dns\*.cs;..\Google.PowerShell\Properties\*.cs;..\Google.PowerShell\Provider\*.cs;..\Google.PowerShell\Sql\*.cs;..\Google.PowerShell\Storage\*.cs;..\Google.PowerShell\Logging\*.cs;..\Google.PowerShell\PubSub\*.cs;..\Google.PowerShell\BigQuery\*.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**" />
+    <Compile Include="..\Google.PowerShell\**\*.cs" Exclude="..\**\bin\**;..\**\obj\**;**\*.xproj;packages\**;..\Google.PowerShell\CloudResourceManager\IamPolicyCmdlets.cs;..\Google.PowerShell\Container\*" />
```

Comment:
Does ..\Google.PowerShell\**\*.cs cover the CloudResourceManager and Container too? If so, then I think we can remove those too.

Reply:
The exclude excludes Container and the one cmdlet set that is problematic in CloudResourceManager.